configure

```diff
@@ -1066,7 +1066,7 @@ echo "#define OCAML_STDLIB_DIR \"$libdir\"" >> s.h
 
 # Do #! scripts work?
 
-if (SHELL=/bin/sh; export SHELL; (./hashbang || ./hashbang2 || ./hashbang3) >/dev/null); then
+if ((./hashbang || ./hashbang2 || ./hashbang3 || printf "#!%s\nexit 1\n" `command -v cat` > hashbang; ./hashbang) >/dev/null); then
```

Comment:
I didn't want to introduce a call to `chmod`, but sure, can generate a fresh file